config/database.yml

```diff
@@ -17,6 +17,8 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  username: <%= ENV["POSTGRES_USERNAME"] %>
+  password: <%= ENV["POSTGRES_PASSWORD"] %>
```

Comment:
hey @aonomike ... I know these are different from the values you use, I'm open to change them, if you want

Reply:
I think I like them